--- node_modules/boost-lib/cmake/BoostLib.cmake	2016-09-19 21:47:52.000000000 +0300
+++ node_modules/boost-lib/cmake/BoostLib.cmake	2016-09-22 21:37:05.329262741 +0300
@@ -24,7 +24,7 @@
     message(STATUS "Boost Libraries: ${Boost_LIBRARIES}")
 
     # Results:
-    set(Boost_FOUND ${Boost_FOUND} PARENT_SCOPE)
-    set(Boost_INCLUDE_DIRS "${Boost_INCLUDE_DIRS}" PARENT_SCOPE)
-    set(Boost_LIBRARIES "${Boost_LIBRARIES}" PARENT_SCOPE)
-endfunction(require_boost_libs)
\ No newline at end of file
+    set(Boost_FOUND ${Boost_FOUND} CACHE BOOLEAN "" FORCE)
+    set(Boost_INCLUDE_DIRS "${Boost_INCLUDE_DIRS}" CACHE STRING "" FORCE)
+    set(Boost_LIBRARIES "${Boost_LIBRARIES}" CACHE STRING "" FORCE)
+endfunction(require_boost_libs)
